README.md

```diff
@@ -55,110 +55,89 @@ To use the SQL Server connector, follow these steps in a Hasura project:
 (Note: for more information on the following steps, please refer to the Postgres connector documentation [here](https://hasura.io/docs/3.0/getting-started/connect-to-data/connect-a-source))
 
 ### 1. Init the connector
-(Note: here and following we are naming the subgraph "my_subgraph" and the connector "my_sql")
+(Note: here and following we are naming the subgraph "my_subgraph" and the connector "ms_sql")
 
    ```bash
-   ddn connector init my_sql --subgraph my_subgraph --hub-connector hasura/sqlserver
+   ddn connector init ms_sql --subgraph my_subgraph/subgraph.yaml --hub-connector hasura/sqlserver --configure-port 8081 --add-to-compose-file compose.yaml
    ```
 
 ### 2. Add your SQLServer credentials
 
-Add your credentials to `my_subgraph/connector/my_sql/.env.local`
+Add your credentials to `my_subgraph/connector/ms_sql/.env.local`
 
-```env title="my_subgraph/connector/my_sql/.env.local"
+```env title="my_subgraph/connector/ms_sql/.env.local"
 OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://local.hasura.dev:4317
-OTEL_SERVICE_NAME=my_subgraph_my_sql
+OTEL_SERVICE_NAME=my_subgraph_ms_sql
 CONNECTION_URI=<YOUR_SQLSERVER_URL>
 ```
 
-### 3. Introspect your indices
+### 3. Introspect your Database
 
 From the root of your project run:
 
 ```bash title="From the root of your project run:"
-ddn connector introspect --connector my_subgraph/connector/my_sql/connector.yaml
+ddn connector introspect --connector my_subgraph/connector/ms_sql/connector.local.yaml
 ```
 
 If you look at the `configuration.json` for your connector, you'll see metadata describing your SQL Server mappings.
 
-### 4. Create the Hasura metadata
+### 4. Restart the services
 
-Run the following from the root of your project:
+Let's restart the docker compose services. Run the folowing from the root of your project:
 
-```bash title="Run the following from the root of your project:"
-ddn connector-link add my_sql --subgraph my_subgraph
-```
-
-The generated file has two environment variables — one for reads and one for writes — that you'll need to add to your
-subgraph's `.env.my_subgraph` file. Each key is prefixed by the subgraph name, an underscore, and the name of the
-connector. Ensure the port value matches what is published in your connector's docker compose file.
-
-```env title="my_subgraph/.env.my_subgraph"
-MY_SUBGRAPH_MY_SQL_READ_URL=http://local.hasura.dev:8081
-MY_SUBGRAPH_MY_SQL_WRITE_URL=http://local.hasura.dev:8081
+```bash title="From the root of your project run:"
+HASURA_DDN_PAT=$(ddn auth print-pat) docker compose up --build --watch
 ```
 
-### 5. Start the connector's docker compose
+The schema of the database can be viewed at http://localhost:8081/schema.
```

Comment:
Let's not have this line here, because the schema at this point will be an empty schema. Let's move this step after we do `ddn connector-link update` so that the user can check the schema corresponding to their database.

Reply:
Done